tests/test_types.py

```diff
@@ -1258,3 +1258,514 @@ class JsonDetailedModel(BaseModel):
         'msg': 'JSON object must be str, bytes or bytearray',
         'type': 'type_error.json'
     }
+
+
+@pytest.mark.parametrize(
```

Comment:
new file i think.

Also, are these taken from marshmallow or similar? If so I think we should attribute the code/cases.

Reply:
moved to a new file